### PR TITLE
Adds <= and >= Math operators to applicative

### PR DESCRIPTION
**Pull request by MajorZiploc**

### CODE REVIEW

**Comment by gusty** [comment]
Thanks !
Could you add some tests, at least basic ones?

**Comment by MajorZiploc** [comment]
Yes, of course. I was having some trouble building the test suite. I will look into it tomorrow and and some tests.

**Comment by gusty** [comment]
Great, let us know if you can't solve the problems with the tests.

File: tests/FSharpPlus.Tests/General.fs [hunk 1]
```diff
@@ -1162,6 +1163,29 @@ module Applicative =
         Assert.AreEqual ([113; 213; 123; 223; 114; 214; 124; 224], testVal)
         Assert.IsInstanceOf<Option<list<int>>> (Some testVal)
 
+        let testLTE1 = (Some 1) .<=. (Some 2)
```
Comment (gusty) [inline tests/FSharpPlus.Tests/General.fs:1166]: Do we really need enclosing parens here?
I think the operator alone should be enough.

**Pull request merged**